todo/src/reducer.js

```diff
@@ -0,0 +1,23 @@
+import { ADD_TASK, DECREMENT } from '../actions';
+
+const initialState = {
+  todo: []
+}
```

Comment:
Good job getting the Redux store, reducers, and actions setup. Now build out your components and connect it all together!